scipy/sparse/linalg/isolve/tests/test_minres.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import division, print_function, absolute_import
+
+import numpy as np
+from numpy.testing import (assert_equal, assert_allclose, assert_,
+                           run_module_suite, dec)
```

Comment:
`run_module_suite` (just remove the `if __name__ == "__main__":` part) and `dec` shouldn't be used any more. Instead of `dec`, use pytest's native stuff.